### ISSUE TITLE
AbortController not available for Aurelia template

### ISSUE THREAD

**Issue by dackjaniels2001** [RicoSuter/NSwag#3452]
Hi,

A previous issue discussed adding the AbortController option to the Fetch template of Nswag studio.
Could this option also be made available for the Aurelia template? I would like to use it but it's not currently available.

Steve

**Comment by RicoSuter** [comment]
> UPDATE: I just worked around this by choosing Fetch, checking the AbortController box and then selecting Aurelia again and generating the files. The Aurelia template was used but also included the optional signal parameter for the abort functionality.

So the option works but is just hidden in the UI (NSwagStudio)?

**Comment by dackjaniels2001** [comment]
Yes, it's just hidden when the Aurelia template is selected but still
appears to work.
